### PR TITLE
Adding the range of motion (max. orientation change) to the list of spatial parameters.

### PR DESCRIPTION
**Pull request by AKuederle**

### CODE REVIEW

File: gaitmap/parameters/_spatial_parameters.py [hunk 1]
```diff
@@ -93,6 +94,9 @@ class SpatialParameterCalculation(BaseSpatialParameterCalculation):
     Arc Length
         The overall arc length is directly calculated from the position of the sensor by adding the absolute changes in
         position at every time point.
+    Max. orientation change
```
Comment (AKuederle) [inline gaitmap/parameters/_spatial_parameters.py:97]: I like the naming! Good to differentiate it from range-of-motion :)

**Comment by AKuederle** [comment]
@HaMora Any updates on this?

**Comment by HaMora** [comment]
Hi @AKuederle,
Can we have a short talk about this next week? I really wanna know the underlying work for the tests and the snapshots that we have. 
As far as I understand, the test fails since we need a new snapshot with the new parameter. Which, honestly; I didn't spend much time as well to figure out how to create them :)

**Comment by HaMora** [comment]
Hi @AKuederle, 
I have fixed the tests and replaced the snapshots (thank you for the explanations).
I just don't know why the docs test failed.

File: gaitmap/parameters/_spatial_parameters.py [hunk 2]
```diff
@@ -93,6 +94,9 @@ class SpatialParameterCalculation(BaseSpatialParameterCalculation):
...
+        The maximum change of angle in the sagittal plane. It is similar to the range of motion, however, the measured
+        parameter is value effected by other joints such as knee and hip as well.
```
Comment (AKuederle) [inline gaitmap/parameters/_spatial_parameters.py:99]: Might be a grammer error here.

**Review by AKuederle — COMMENTED** [review]
Looks great to me @HaMora Can you add a Changelog entry (see CHANGELOG.md). Just add a new secition called unreleased at the top, quickly describe the change and add a link to this PR

File: gaitmap/parameters/_spatial_parameters.py (hunk 2 above)
Comment (AKuederle) [reply gaitmap/parameters/_spatial_parameters.py]: Took care of it, as I had to do some other gaitmap stuff as well and want to get a release done today :)

Thanks for your work on this!

**Pull request merged**